spec.html

```diff
@@ -427,7 +427,7 @@ <h1>Infinity</h1>
 
     <emu-clause id="sec-terms-and-definitions-nan">
       <h1>NaN</h1>
-      <p>Number value that is an IEEE 754-2019 “Not-a-Number” value</p>
+      <p>Number value that is an IEEE 754-2019 “Not-a-Number” (NaN) value</p>
```

Comment:
```suggestion
      <p>Number value that is an IEEE 754-2019 NaN ("not a number")</p>
```

Current wording implies that IEEE calls these "Not-a-Number values", which it doesn't. It consistently refers to "NaNs".